### PR TITLE
build: publish@4.2.0-beta.0

### PR DESCRIPTION
**Pull request by ZLY201**
Publish version 4.2.0-beta.0.

Expected to be released at 10.30.

**note: The last commit must be build: publish@{version}.**

CheckList:

- [x] check version is valid
- [x] check all the changes are update in change log
- [x] check all the version tags in `json` have been changed
- [x] check all funding links in `.github/FUNDING.yml` are valid
- [x] check push `main` workflow syntax and build scripts are valid
- [x] check documentation is changed according to `pr-preview`

### CODE REVIEW

**Comment by ZLY201** [comment]
`Compressed Size Compare` is failed because the `main` branch doesn't have the `build:all` command. It will be auto fixed in next version.

**Comment by ZLY201** [comment]
Is there any change these day you want to push?

If not I'll lock a version for `dev` branch and please help to check with`CheckList`.

@noname0310

File: conf/isolate.config.json [hunk 1]
```diff
@@ -3,72 +3,52 @@
     {
       "name": "stack",
       "version": "4.1.5",
```
Comment (noname0310) [inline conf/isolate.config.json:5]: it might be `"version": "4.2.0-beta.0"`

**Comment by noname0310** [comment]
> Is there any change these day you want to push?
> 
> If not I'll lock a version for `dev` branch and please help to check with`CheckList`.
> 
> @noname0310

The only thing I'm working on right now is #93 and this can be handled later.

**Review by noname0310 — DISMISSED** [review]
There seems to be no problem with the build script. Everything looks fine

File: conf/isolate.config.json (hunk 1 above)
Comment (ZLY201) [reply conf/isolate.config.json]: Done.

**Comment by ZLY201** [comment]
`dev` branch locked.

**Review by noname0310 — APPROVED** [review]

**Pull request merged**